doc/man3/SSL_get_error.pod

```diff
@@ -126,6 +126,13 @@ OpenSSL error queue contains more information on the error.
 
 =back
 
+=head1 BUGS
+
+In case of SSL_ERROR_SYSCALL, B<ret> is meant to indicate that an error is
+retryable or not.
```

Comment:
It is never be retryable if you get SSL_ERROR_SYSCALL. SSL_get_error() should give you an SSL_ERROR_WANT_* return if it is retryable. The previous description of SSL_ERROR_SYSCALL is just wrong I think,